.github/workflows/rust.yml

```
@@ -328,7 +328,7 @@ jobs:
       run: |
         cargo fmt -- --check
 
-  lint-cargo-sort:
+  lint-taplo-sort:
```

Would it make sense to have a more generic name?

```suggestion
  lint-check-if-tomls-are-sorted:
```

---

Checking if tomls are sorted is all `taplo fmt` does, AFAIU 🤔
And this is the naming we're following on the other jobs, so if we change this one to be more verbose I think we have to change all others also, probably